updating/updating-cluster.adoc

```diff
@@ -7,6 +7,18 @@ toc::[]
 
 You can update, or upgrade, an {product-title} cluster.
 
+[IMPORTANT]
+====
+During the {product-title} beta period, update functionality is limited:
+
+* You cannot update between different beta drops, such as from OpenShift 4 Beta
+Drop 2 to OpenShift 4 Beta Drop 3.
+* The ability to update from a beta drop of {product-title} to a full version of
+is not guaranteed.
+* After you install OpenShift 4 Beta Drop 3, you can update your cluster to the
+available nightly builds.
```

Comment:
`available nightly builds` ?
@sdodson do you know if it is any nightly builds after beta3 for some specific builds ?
also how many night builds all / first x / just next one ?

Reply:
I believe it will be a set of specifically tested builds.

Reply:
From the customer's perspective they wouldn't know that, they'll only see the updates we want them to.

Reply:
Right. I though I heard that it was a set of three validated daily builds, so from a customer's perspective, it'll look like nightly builds. I'm happy to change the text if you think something else would be less confusing for both internal and external audiences.

Reply:
I think it's fine as written.